bundle/edu.gemini.pit/src/main/java/edu/gemini/pit/ui/util/SharedIcons.java

```diff
@@ -52,21 +56,21 @@ public void paintIcon(Component c, Graphics g, int x, int y) { }
     Icon ICON_SPINNER = new SharedIcon("spinner.gif");
     Icon ICON_SPINNER_BLUE = new SharedIcon("spinner_blue_gray.gif");
 
-	Icon ICON_SIDEREAL = new SharedIcon("sidereal.png");
+    SharedIcon ICON_SIDEREAL = new SharedIcon("sidereal.png");
     Icon ICON_SIDEREAL_DIS = new SharedIcon("sidereal_dis.png");
-	Icon ICON_NONSIDEREAL = new SharedIcon("nonsidereal.png");
-	Icon ICON_TOO = new SharedIcon("too.png");
+    SharedIcon ICON_NONSIDEREAL = new SharedIcon("nonsidereal.png");
+    SharedIcon ICON_TOO = new SharedIcon("too.png");
```

Comment:
I think that is a leftover of a bad experiment when I was trying to use icons on the radio buttons, will be removed